README.md

```diff
@@ -34,23 +57,54 @@ go get github.com/libp2p/go-libp2p-pubsub
 
 To be used for messaging in p2p instrastructure (as part of libp2p) such as IPFS, Ethereum, other blockchains, etc.
 
-## Implementations
+### Examples
 
-See [libp2p/specs/pubsub#Implementations](https://github.com/libp2p/specs/tree/master/pubsub#Implementations).
+`To be written`
 
 ## Documentation
 
-See the [libp2p specs](https://github.com/libp2p/specs/tree/master/pubsub) for high level documentation
-and [godoc](https://godoc.org/github.com/libp2p/go-libp2p-pubsub) for API documentation.
+See the [libp2p specs](https://github.com/libp2p/specs/tree/master/pubsub) for high level documentation and [godoc](https://godoc.org/github.com/libp2p/go-libp2p-pubsub) for API documentation.
 
+### In this repo, you will find
```

Comment:
Is this really useful?

Reply:
I believe it is. Have used it in other projects to lay the map of the land for newcomers. It reduces the "daunting factor" when arriving at a unknown codebase

Reply:
I would order the subsystem implementation files first and the routers last.